src/lib/convenience/UpdateContextElementResponse.cpp

```diff
@@ -148,7 +148,7 @@ void UpdateContextElementResponse::fill(UpdateContextResponse* ucrsP)
     //
     for (unsigned int aIx = 0; aIx < cerP->contextElement.contextAttributeVector.size(); ++aIx)
     {
-      cerP->contextElement.contextAttributeVector[aIx]->value = "";
+      cerP->contextElement.contextAttributeVector[aIx]->stringValue = "";  // FIXME P10: automacit value -> stringValue change, please review to check if it is safe
```

Comment:
Maybe adding a comment about it (i.e. that given that this is strictly /v1 it isn't a problem to ignore other value fields appart of stringValue) would be a good idea.